_data/simple-icons.json

```diff
@@ -11253,6 +11253,16 @@
             "source": "https://en.facebookbrand.com/oculus/assets/oculus?audience=oculus-landing",
             "guidelines": "https://en.facebookbrand.com/oculus/"
         },
+        {
+            "title": "Odigos",
+            "hex": "F9F9F9",
+            "source": "https://odigos.io/icons/brand",
+            "aliases": {
+                "aka": [
+                    "keyval"
+                ]
+            }
```

Comment:
Sorry, I don't think `keyval` is a well-known aliases name for this project.

Reply:
OK, I removed it 👍